app/controllers/schedule_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class ScheduleController < ApplicationController
+  def save_scheduled_publishing_datetime
+    Document.transaction do
+      document = Document.with_current_edition.lock.find_by_param(params[:id])
+      @edition = document.current_edition
+
+      checker = Requirements::ScheduledDatetimeChecker.new(permitted_params)
+      date_issues = checker.date_issues
```

Comment:
Could we just have a `pre_submit_issues` method that collects these? We have this pattern in some of the other checkers - this means you can then do `if issues.any?` and `issues.items` :-).

Reply:
Just thought of another advantage of doing this - it would help get rid of the duplication of `return unless date_issues.items.empty? && time_issues.items.empty?` :-).

Reply:
Done